joss/paper.md

```diff
@@ -76,7 +75,7 @@ Installation instructions, tutorials, API reference, and open-source code are av
 
 ![Contextualized Machine Learning paradigm.\label{fig:paradigm}](figs/context_encoders_sideways.pdf){width=100%}
```

Comment:
Can this be moved onto the first page, if the above paragraph is only written in "one-stop shop" section?